packages/graphiql/src/components/GraphiQL.tsx

```diff
@@ -1490,7 +1490,7 @@ export class GraphiQL extends React.Component<GraphiQLProps, GraphiQLState> {
     // Use the edited query after autoCompleteLeafs() runs or,
     // in case autoCompletion fails (the function returns undefined),
     // the current query from the editor.
-    const editedQuery = this.autoCompleteLeafs() || this.state.query;
+    const editedQuery = this.autoCompleteLeafs() || this.state.query || '';
```

Comment:
`editedQuery` _has_ to be a string because `_fetchQuery` does `query +=` which will only work well if it's already a string. In all the other cases, `undefined` is handled just fine hence the type change to the private `_fetchQuery` parameters.

Reply:
good find!